crates/brioche/src/format.rs

```diff
@@ -26,11 +26,17 @@ pub async fn format(args: FormatArgs) -> anyhow::Result<ExitCode> {
     for project_path in args.project {
         match project_format(&reporter, &project_path, args.check).await {
             Err(err) => {
-                println!(
-                    "Error occurred while formatting project '{project_path}': {err}",
-                    project_path = project_path.display(),
-                    err = err
-                );
+                reporter.emit(superconsole::Lines::from_multiline_string(
+                    &format!(
+                        "Error occurred while formatting project '{project_path}': {err}",
+                        project_path = project_path.display(),
+                        err = err
+                    ),
+                    superconsole::style::ContentStyle {
+                        foreground_color: Some(superconsole::style::Color::Red),
```

Comment:
Should I leave it red ? I’ll be inconsistent with the other commands I think. Unless we try to make use of the reporter everywhere else.